src/components/ScheduleCard/index.js

```diff
@@ -1 +1,28 @@
 import React from 'react';
+
+export const ScheduleCard = ({ booking, type, cancelPairing }) => {
+  const { notes, pairee, pairer, time, date } = booking;
+  return (
+    <div className={type === 'pairer' ? 'pairer' : 'pairee'}>
+      <p>{date}</p>
+      {time === 'morning' && <p>8:00 - 8:50 a.m.</p>}
+      {time === 'lunch' && <p>12:00 - 12:50 p.m.</p>}
+      {time === 'afternoon' && <p>4:00 - 4:50 p.m.</p>}
+      {type === 'pairer' && (
+        <p>
+          {pairee.name} from {pairee.program} Mod {pairee.module} signed up to
+          pair with you.
+        </p>
+      )}
+      {type === 'pairee' && <p>You signed up to pair with {pairer.name}.</p>}
+      {type === 'pairer' && (
+        <p>
```

Comment:
Great work here with showing either the notes about the booking or the pairee's slack so the pairer can reach out if they want to prepare ahead of time